docs/how-to/artifact.rst

```diff
@@ -20,8 +23,62 @@ corresponds to the ``created_at`` attribute.
 
 .. important::
 
-    ``lazyscribe`` does not create the directory for you. You will need to use
-    :py:meth:`pathlib.Path.mkdir`.
+  Unless you log an artifact, this directory will not be created automatically.
+
+To save an artifact to this directory, use :py:meth:`lazyscribe.Experiment.log_artifact`.
+Serialization is delegated to a subclass of :py:class:`lazyscribe.artifacts.Artifact`.
+For example, you can persist a ``scikit-learn`` estimator using ``joblib``:
+
+.. code-block:: python
+    :emphasize-lines: 8-9
+
+    from lazyscribe import Project
+    from sklearn.svm import SVC
+
+    project = Project()
+    with project.log("My experiment") as exp:
+        X, y = ...
+        model = SVC()
+        model.fit(X, y)
+        exp.log_artifact(fname="estimator.joblib", value=model, handler="scikit-learn")
+
+Below, we have included a list of currently supported artifact handlers and their aliases:
+
+.. list-table:: Builtin artifact handlers
+    :header-rows: 1
+
+    * - Class
+      - Alias
+      - Description
+    * - :py:class:`lazyscribe.artifacts.JSONArtifact`
+      - json
+      - Artifacts written using :py:meth:`json.dump` and read using :py:meth:`json.load`
+    * - :py:class:`lazyscribe.artifacts.SklearnArtifact`
+      - scikit-learn
+      - Artifacts written using :py:meth:`joblib.dump` and read using :py:meth:`joblib.load`
```

Comment:
It may be obvious for these two handlers, but should we add an additional column to this table listing the additional required packages?